sale_packaging_default/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Default packaging for sales",
-    "version": "16.0.2.0.0",
+    "version": "16.0.2.1.0",
```

Comment:
Same remark here.
Change to this module should not be part of this PR but in another separate one